pyppeteer/coverage.py

```diff
@@ -196,16 +199,19 @@ def _onExecutionContextsCleared(self, event: Dict) -> None:
             raise PageError('JSCoverage is not enabled.')
         self._enabled = False
 
-        result = await self._client.send('Profiler.takePreciseCoverage')
-        await self._client.send('Profiler.stopPreciseCoverage')
-        await self._client.send('Profiler.disable')
-        await self._client.send('Debugger.disable')
+        result = await asyncio.gather(
+            self._client.send('Profiler.takePreciseCoverage'),
+            self._client.send('Profiler.stopPreciseCoverage'),
+            self._client.send('Profiler.disable'),
+            self._client.send('Debugger.disable'),
+        )
         helper.removeEventListeners(self._eventListeners)
 
         coverage: List = []
```

Comment:
mypy is smart enough to infer this type 😉 
```suggestion
        coverage = []
```